testing/cloud.py

```diff
@@ -47,6 +47,20 @@ def __getattr__(self, name):
         return os.environ[name]
 
 
+def get_resource_path(resource, local_path):
```

Comment:
Hmm. I guess I could make it the first argument if you want - that would more or less fix it.